packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -694,7 +694,7 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     const model = modelNameToModelDefinition[modelName];
 
     resolverObj[relationOwner] = (parent: any, args: any, context: GraphbackContext, info: GraphQLResolveInfo) => {
-      if (parent[relationOwner]) {
+      if (relationOwner in parent) {
```

Comment:
The `in` key can lead to false positives or negatives:

```ts
'constructor' in obj; // true
'__proto__' in obj; // true
'hasOwnProperty' in obj; // true

obj.hasOwnProperty('constructor'); // false
obj.hasOwnProperty('__proto__'); // false
obj.hasOwnProperty('hasOwnProperty'); // false
```

I can't see this being a problem for the `parent` arg, but wanted to highlight it in case.

https://masteringjs.io/tutorials/fundamentals/hasownproperty#:~:text=The%20key%20difference%20is%20that,property%2C%20and%20a%20hasOwnProperty%20function.

However, `hasOwnProperty` is not available in TS yet, another solution is to make our own: https://fettblog.eu/typescript-hasownproperty/

Reply:
Really good point. 

`hasOwnProperty in obj` is an oversight from me. 

Let me fix that.